server/broker/broker.go

```diff
@@ -0,0 +1,26 @@
+//Copyright 2017 Huawei Technologies Co., Ltd
```

Comment:
ServiceComb already moved in Apache infra, please update the lisence.

Reply:
done.